paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/LinkTest.kt

```diff
@@ -212,7 +212,7 @@ internal class LinkTest {
             networkRule.enqueue(
                 method("POST"),
                 path("/v1/payment_intents/pi_example/confirm"),
-                bodyPart(urlEncode("payment_method_options[card][setup_future_usage]"), "off_session"),
+                bodyPart(urlEncode("payment_method_options[link][setup_future_usage]"), "off_session"),
```

Comment:
This test is using payment method mode, which means we should be using `link` instead of `card` here.